ibm_catalog.json

```diff
@@ -0,0 +1,251 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-vpc-private-path",
+      "label": "Cloud automation for VPC Private Path",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "solution",
+        "security"
```

Comment:
I don't think falls under the `security` category. I suggest to use `network`. For full list of tags you can run `ibmcloud catalog filter options --all`